apps/src/code-studio/projectRedux.js

```diff
@@ -0,0 +1,125 @@
+/** Redux actions and reducer for the inidividual projects */
+/* globals dashboard */
+
+const SHOW_PROJECT_UPDATED_AT = 'header/SHOW_PROJECT_UPDATED_AT';
+const SET_PROJECT_UPDATED_STATUS = 'header/SET_PROJECT_UPDATED_STATUS';
+const SET_PROJECT_UPDATED_AT = 'header/SET_PROJECT_UPDATED_AT';
+const REFRESH_PROJECT_NAME = 'header/REFRESH_PROJECT_NAME';
+const SHOW_TRY_AGAIN_DIALOG = 'header/SHOW_TRY_AGAIN_DIALOG';
+const SET_NAME_FAILURE = 'header/SET_NAME_FAILURE';
+const UNSET_NAME_FAILURE = 'header/UNSET_NAME_FAILURE';
```

Comment:
these actions also still use the "header/" prefix. in both cases, the prefix should match the reducer name, so here the prefix would be "project/"

Reply:
Thanks! Will do.